test/core/bin.js

```diff
@@ -132,7 +132,12 @@ _.each(fs.readdirSync(fixtureLoc), function (binName) {
       };
 
       var optionsLoc = testLoc + "/options.json"
-      if (fs.existsSync(optionsLoc)) _.merge(opts, require(optionsLoc));
```

Comment:
This is going to break the test harness since it needs all options to be JSON since it's serialised for the browser tests. Probably best to manually put this as an [API](https://github.com/babel/babel/blob/master/test/core/api.js) test.